atom/browser/native_window_views_win.cc

```diff
@@ -90,6 +90,15 @@ bool NativeWindowViews::PreHandleMSG(
     // mode if it isn't already, always say we didn't handle the message
     // because we still want Chromium to handle returning the actual
     // accessibility object.
+    case WM_DWMCOLORIZATIONCOLORCHANGED: {
```

Comment:
Putting this in `NativeWindowViews` makes the event get emitted for multiple times when there are multiple windows. The common way is to create [message window](https://msdn.microsoft.com/en-us/library/windows/desktop/ms632599(v=vs.85).aspx#message_only) to receive the messages, and there is a helper class in Chromium to do it: https://cs.chromium.org/chromium/src/base/win/message_window.h?sq=package:chromium.

By using it we can also move the API to `systemPreferences`.